paper.md

```diff
@@ -0,0 +1,74 @@
+
+CLOVER (Continuous Lifetime Optimisation of Variable Electricity Resources) is a software tool for simulating and optimising community-scale energy systems, typically minigrids, to support energy access in developing countries. CLOVER can be used to model electricity demand and supply at an hourly resolution, for example allowing users to investigate how an electricity system might perform at a given location. CLOVER can also identify an optimally-sized energy system to meet the needs of the community under specified constraints, for example a system which provides a desired level of reliability at the lowest cost of electricity. CLOVER can provide an insight into the technical performance, costs, and environmental impact of a system, and allow the user to evaluate many different scenarios to decide on the best way to provide sustainable, affordable and reliable electricity to a community.
+
+CLOVER can be used on both personal computers and high-performance computing facilities. Its design separates its general framework (code) from user inputs (data) specific to their investigations, which are input via `YAML` files. Its straightforward command-line interface provides simple operation for both experienced Python users and those with little prior exposure to coding.
+
+# Statement of need
+
+CLOVER was developed to provide a robust techno-economic and environmental analysis tool to support academics, practitioners and policymakers in evaluating sustainable energy interventions in developing countries. By providing an open-source alternative to proprietary minigrid design tools such as HOMER [@HOMER], CLOVER offers a more comprehensive analysis tool than common spreadsheet-based alternatives. CLOVER was designed to provide a fast, flexible and user-friendly software to assess energy systems at both short-term (hourly) and multi-year timescales under a wide variety of user-defined scenarios, constraints and optimisation conditions.
+
+CLOVER can provide an insight into the technical performance, costs, and environmental impact of a system, and allow the user to evaluate many different options to decide on the best way to provide sustainable, affordable and reliable electricity to the community. Its simulation functionality offers the user an insight into the hourly, daily, seasonal and multi-year technical performance of a system in response to changing electricity demands and varying renewable resources. Its optimisation process replicates the design process common in sustainable development settings by identifying optimum system design over short-term time horizons, but allowing the system to increase in capacity over its lifetime as the demand of the community grows in response to its economic development [@Sandwell:2017a].
```

Comment:
I wonder if environmental impact is too broad and GHG impact/climate impact is better given that's the only aspect it can include at present.

Reply:
Agreed - `solar-thermal` and `dev` bring in changes to look at waste products, such as brine or perhaps even CFCs, but this isn't yet released

Reply:
Agreed, I've made "environmental impact" more specific as suggested.